packages/api/src/services/notification-service/notification-service.ts

```diff
@@ -0,0 +1,92 @@
+import { singleton } from 'tsyringe';
+import { DatabaseService } from '../database-service/database-service';
+import { Knex } from 'knex';
+import { NotificationSubscription } from 'common/models/notification-subscription';
+import {
+    sendNotification,
+    PushSubscription,
+    VapidKeys,
+    setVapidDetails,
+} from 'web-push';
+
+@singleton()
+export class NotificationService {
+    readonly vapidKey: VapidKeys = {
+        publicKey:
+            'BC7J1hDqhmWAb-N88JCvA_vgasQ433M8sD4Fl6qWyfiinOgW3qh62vJfrs2wWLg6I26JdsXwoXX2u7YS13XQbJA',
+        privateKey: 'A1Nj53Yi95sVuPCN_xpUjyCnf13xwKh03mf2tXj98VQ',
```

Comment:
Where are those keys from? Could we put them in the `.env` file also?

Reply:
I created them using a generator. They are used to authenticate that the messages sent by us to the client are really ours. I will put them in a .env file.